### PR TITLE
Backport of 66995: mysql_user doesnt support privileges with underscore

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Backport of #66995 : mysql_user doesnt support privileges with underscore

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
```lib/ansible/modules/database/mysql/mysql_user.py```

### CODE REVIEW

**Comment by Andersson007** [comment]
@nitzmahone i reran the tests, so feel free to merge

File: lib/ansible/modules/database/mysql/mysql_user.py [hunk 1]
```diff
@@ -227,15 +230,18 @@
...
+                         'BINLOG_ENCRYPTION_ADMIN', 'CLONE_ADMIN', 'CONNECTION_ADMIN',
+                         'ENCRYPTION_KEY_ADMIN', 'FIREWALL_ADMIN', 'FIREWALL_USER',
+                         'GROUP_REPLICATION_ADMIN', 'INNODB_REDO_LOG_ARCHIVE',
+                         'NDB_STORED_USER', 'PERSIST_RO_VARIABLES_ADMIN',
+                         'REPLICATION_APPLIER', 'REPLICATION_SLAVE_ADMIN',
+                         'RESOURCE_GROUP_ADMIN', 'RESOURCE_GROUP_USER',
+                         'ROLE_ADMIN', 'SESSION_VARIABLES_ADMIN', 'SET_USER_ID',
+                         'SYSTEM_USER', 'SYSTEM_VARIABLES_ADMIN', 'SYSTEM_USER',
+                         'TABLE_ENCRYPTION_ADMIN', 'VERSION_TOKEN_ADMIN',
+                         'XA_RECOVER_ADMIN', 'LOAD FROM S3', 'SELECT INTO S3'))
```
Comment (mattclay) [inline lib/ansible/modules/database/mysql/mysql_user.py]: @Andersson007  Does `'LOAD FROM S3', 'SELECT INTO S3'` belong in the backport? It isn't in the original frozenset from stable-2.8, only the one in devel.

I noticed the same issue in the 2.9 backport, but it has already been merged.

Comment (Andersson007) [reply lib/ansible/modules/database/mysql/mysql_user.py]: @mattclay , public condemnation on me! thanks! Fixed.. PTAL

Comment (Andersson007) [reply lib/ansible/modules/database/mysql/mysql_user.py]: and i wrote a revert PR for 2.9 too https://github.com/ansible/ansible/pull/67955 please take a look

Comment (mattclay) [reply lib/ansible/modules/database/mysql/mysql_user.py]: @Andersson007 Thanks for the quick fix here and for 2.9.

Comment (Andersson007) [reply lib/ansible/modules/database/mysql/mysql_user.py]: @mattclay , sure, it was my mistake, thanks for pointing it out!

Comment (Andersson007) [reply lib/ansible/modules/database/mysql/mysql_user.py]: @mattclay , sure, it was my mistake, thanks for pointing it out!

**Pull request merged**